spec/unit/command/verify_spec.rb

```diff
@@ -280,10 +280,10 @@ def stdout
           expect(stdout).to include("my friend everything is good...")
         end
 
-        it "should report the output of the first verification first" do
+        it "should report the output of the first verification first", :volatile => true do
           index_first = stdout.index("you are good to go...")
           index_second = stdout.index("my friend everything is good...")
-          expect(index_second > index_first).to be true
+          expect(index_second).to be > index_first
```

Comment:
Does RSpec issue a warning for this? I know that they disallow or discourage the use of `expect(thing).to == other_thing` and also `expect(array_thing).to =~ other_array_thing` (for when you just want all the elements to be present but don't care about the order) and have alternatives, i.e., `expect(thing).to eq(other)` and `expect(thing).to array_match(other)`.